src/midiEditorCommands.cpp

```diff
@@ -2292,6 +2293,94 @@ void postToggleFunctionKeysAsStepInput(int command) {
 	}
 }
 
+void postMidiToggleMute(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int eventCount = countSelectedEvents(take);
+	if (eventCount == 0) {
+		return;
+	}
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	// Get selected CCs.
+	vector<MidiControlChange> selectedCCs = getSelectedCCs(take);
+	int noteCount = selectedNotes.size();
+	int CCCount = selectedCCs.size();
+	ostringstream s;
+	if (CCCount == 0) { // If only notes are selected
+		auto chord = findChord(take, 0, {
+			true,  // start
+			true,  // end
+			true,  // channel
+			true,  // pitch
+			true,  // velocity
+			false,  // selected
+			true  // muted
+		});
+		// Select and play the current chord.
+		vector<MidiNote> notes(chord.first, chord.second);
+		previewNotes(take, notes);
+		fakeFocus = FOCUS_NOTE;
+		if (settings::reportNotes) {
+			if (noteCount == 1) {
+				if (selectedNotes[0].muted)
+				s << translate("muted") << " ";
+				else
+				s << translate("unmuted") << " ";
```

Comment:
1. These two `s <<` lines should be indented 1 additional level.
2. I'd suggest bracing the if and else blocks here. Even though there's only 1 line of code each, it can be confusing, particularly in a nested if statement and particularly if the indentation gets messed up.